logic/src/commonMain/kotlin/com/wire/kalium/logic/data/notification/LocalNotification.kt

```diff
@@ -42,8 +42,7 @@ sealed class LocalNotificationMessage(
         override val author: LocalNotificationMessageAuthor,
         override val time: Instant,
         val text: String,
-        val isQuotingSelfUser: Boolean = false,
-        val isMentionedSelf: Boolean = false
+        val isQuotingSelfUser: Boolean = false
```

Comment:
can we cover this case with tests to avoid breaking it in the future ?

Reply:
good point :)